packages/runtime/datastore-definitions/api-report/datastore-definitions.api.md

```diff
@@ -26,7 +26,7 @@ import { ISummaryTreeWithStats } from '@fluidframework/runtime-definitions';
 import { ITelemetryContext } from '@fluidframework/runtime-definitions';
 import { ITelemetryLogger } from '@fluidframework/core-interfaces';
 
-// @alpha (undocumented)
+// @public (undocumented)
```

Comment:
We really need to document this. Probably not for this PR, but we should add a task. This is a pretty core aspect of the framework, and it's undocumented 😕

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6605